.lighthouserc.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
       url: [
         "http://varnish:8080/",
         // TODO: When performance has been improved these URl's can be reactivated:
-        // "http://varnish:8080/search?q=harry+potter&x=0&y=0",
+        "http://varnish:8080/search?q=harry+potter&x=0&y=0",
```

Comment:
Remember to change this before you merge.

Reply:
👍